lib/ringcaptcha/api.rb

```diff
@@ -35,5 +35,16 @@ def self.do_http_request(http, request)
         end
       end
     end
+
+    def self.service_up?
+      uri = URI.parse("https://api.ringcaptcha.com")
+      http = Net::HTTP.new(uri.host, uri.port)
+      http.use_ssl = true
+      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+
+      response = http.request(Net::HTTP::Post.new("/status"))
+
+      response.is_a?(Net::HTTPSuccess) && response.body == "WE ARE ALIVE ;-)"
```

Comment:
Any reason for POST over GET?
Did you consider not checking for the response body? i.e. just 200 OK would be enough.
You could then also avoid pulling the body and use just a more efficient HEAD request.